boards/qemu_rv32_virt/Makefile

```diff
@@ -0,0 +1,7 @@
+# Makefile for building the Tock kernel for the qemu-system-riscv32
+# virt machine type
+
+TARGET=riscv32imac-unknown-none-elf
+PLATFORM=qemu_rv32_virt
+
+include ../Makefile.common
```

Comment:
Can you add the commands to run this here as well?

Reply:
Done.